runner/ansible/roles/1.3.3/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.3.3'
+
+- name: "{{ test_id }}.check"
+  shell: 'systemctl is-enabled sbd || true'
```

Comment:
Could we use the `systemd` module and the `check` mode here?
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html

Reply:
good idea. I will give it a shot.